pilot/pkg/model/push_context.go

```diff
@@ -390,6 +390,9 @@ type PushRequest struct {
 	// Delta defines the resources that were added or removed as part of this push request.
 	// This is set only on requests from the client which change the set of resources they (un)subscribe from.
 	Delta ResourceDelta
+
+	// Forced defines that configs should be generated and pushed regardless if they have changed or not.
+	Forced bool
```

Comment:
Awesome! Can we name it `sidecarScoped`? That way, we don't have to write `Forced: true` everywhere.